wave_cfg_file: "config/wave.json"
ns3_address: "172.2.0.2"
host_ip: "172.2.0.3"
ns3_registration_port: 1515
ns3_listening_port: 2500
ns3_broadcasting_port: 1516
